hoomd/hpmc/compute.py

```diff
@@ -7,123 +7,111 @@
 from __future__ import print_function
 
 from hoomd import _hoomd
+from hoomd.operation import Compute
 from hoomd.hpmc import _hpmc
 from hoomd.hpmc import integrate
-from hoomd.operation import Compute
+from hoomd.data.parameterdicts import ParameterDict
+from hoomd.data.typeconverter import OnlyTypes
+from hoomd.logging import log
 import hoomd
 
-class free_volume(Compute):
-    R""" Compute the free volume available to a test particle by stochastic integration.
+class FreeVolume(Compute):
+    """Compute the free volume available to a test particle by stochastic integration.
```

Comment:
Update the `Args:` list to match the new constructor arguments (see below). Also, add an `Attributes:` section that lists the param_dict entries.